testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -49,168 +42,163 @@
     PSF_NAMD, PDB_NAMD,
     GRO, NUCL, NUCLsel, TPR, XTC,
     TRZ_psf, TRZ,
-    PDB_full,
     PDB_icodes,
 )
 from MDAnalysisTests import make_Universe
 
 import pytest
 
 
-class TestSelectionsCHARMM(TestCase):
-    def setUp(self):
+class TestSelectionsCHARMM(object):
+    @pytest.fixture()
```

Comment:
can we increase the scope of this?  You'll need to check if any test methods modify the universe in their test.  If so, they'l need a function scoped version of the same Universe